artifacts/definitions.py

```diff
@@ -5,25 +5,12 @@
 TYPE_INDICATOR_ARTIFACT_GROUP = 'ARTIFACT_GROUP'
 TYPE_INDICATOR_COMMAND = 'COMMAND'
 TYPE_INDICATOR_DIRECTORY = 'DIRECTORY'
-TYPE_INDICATOR_ENVIRONMENT = 'ENVIRONMENT'
```

Comment:
It doesn't appear to be used anywhere and isn't in the SourceTypeFactory.  I thought this might be cruft from older versions.  I can put it back if it is still valid.